tools/cli/workflow.go

```diff
@@ -82,50 +82,24 @@ func newWorkflowCommands() []cli.Command {
 			Name:    "signal",
 			Aliases: []string{"s"},
 			Usage:   "signal a workflow execution",
-			Flags: []cli.Flag{
-				cli.StringFlag{
-					Name:  FlagWorkflowIDWithAlias,
-					Usage: "WorkflowID",
-				},
-				cli.StringFlag{
-					Name:  FlagRunIDWithAlias,
-					Usage: "RunID",
-				},
-				cli.StringFlag{
-					Name:  FlagNameWithAlias,
-					Usage: "SignalName",
-				},
-				cli.StringFlag{
```

Comment:
Do you intend to make this breaking changes from --input to --signal_input? 
If so you need to put it into the release notes :P 

Or maybe just keep it. I think it makes sense for a signal command to use --input instead of --signal_input because it's already clear that the input is for signal.

Reply:
Good catch, I change the name in the wrong place. Will revert the change here.

Reply:
Reverted to use `--input`